test/fake-server-test.js

```diff
@@ -0,0 +1,31 @@
+(function (root) {
+    "use strict";
+
+    var buster = root.buster || require("buster"),
+        sinon = root.sinon || require("../lib/sinon"),
+        assert = buster.assert,
+        refute = buster.refute;
+
+    buster.testCase("sinon.fakeServer", {
+        ".create": {
+            "allows valid init settings" : function () {
```

Comment:
Try to avoid using words like "valid" and "correct" in test names. Be specific about what your expectations are.

```javascript
"allows 'autoRespond' setting": function() {
```

You should create a test for each of the white listed settings